home.md

```diff
@@ -1,88 +1,25 @@
-<!--
-Les états suivant n'ont été transposé encore avec la réorganisation de la structure du dépôt
-<details>
-	<summary>Définition des états</summary>
-	<dl class="dl-horizontal mrgn-bttm-0">
-		<dt><span class="label label-success">Conforme</span></dt>
-		<dd>Conforme aux exigences définie par la dernière version publié des spécifications du C&amp;AI.</dd>
-
-		<dt><span class="label label-success">Stable</span></dt>
-		<dd>Rencontre les exigences d'une spécification.</dd>
-
-		<dt><span class="label label-info">Information</span></dt>
-		<dd>It's for your information. It's complete and suggestive but not defined by and from a specification yet.</dd>
-
-		<dt><span class="label label-info">Revalidation requise</span></dt>
-		<dd>Précédemment conforme, mais requière une nouvelle validation manuel afin de s'assurer qu'il continue à rencontrer les exigences tel que définie par la dernière version publié de sa spécification.</dd>
-
-		<dt><span class="label label-warning">Partiel</span></dt>
-		<dd>Partiellement conforme au C&amp;AI ou partiellement stable car il inclus des modifications nécessaire afin d'obtenir une conformité aux norme web essentiel, tel que le WCAG 2.0 Niveau AA.</dd>
-
-		<dt><span class="label label-warning">Désuet</span></dt>
-		<dd>N'est pas conforme à la dernière spécification, mais il l'a précedemment été. Son rétablisement à la conformité nécessite des changements immédiat.</dd>
-
-		<dt><span class="label label-default">Arriéré</span></dt>
-		<dd>Nécessite du dévelopement.</dd>
 
-		<dt><span class="label label-danger">Incomplet</span></dt>
-		<dd>Incomplet car il ne se conforme pas complétement à toute les exigences de la spécification. Nécessite encore des travaux de dévelopement.</dd>
-
-		<dt><span class="label label-danger">Obsolète</span></dt>
-		<dd>Ne pas utilisé car c'est obsolète, mais disponible pour votre information.</dd>
-	</dl>
-</details>
--->
-
-<hr />
-
-<div id="english" lang="en">
+{::nomarkdown}
 <div class="row">
 	<div class="col-md-7 col-lg-8">
 		<p>The page templates and design patterns below comprise a reference implementation of the <a href="https://design.canada.ca">Canada.ca design system</a>, including the mandatory requirement of the Content and Information Architecture (C&amp;IA) Specification. Government of Canada departments and agencies can contribute additional patterns and templates via <a href="https://github.com/wet-boew/GCWeb">GCWeb github repository</a>.</p>
 	</div>
 	<div class="col-xs-12 col-md-auto pull-right">
 		<p><a href="https://github.com/wet-boew/GCWeb/archive/v9.3.0.zip" class="btn btn-primary">Download GCWeb theme <strong>v9.3.0</strong></a><br />
```

Comment:
@duboisp We would need to find a way to have the version number and downloadable file to update automatically with every release.